sdl2promela/stop_condition/model.py

```diff
@@ -127,6 +144,9 @@ class NegationExpression(Expression):
     def __init__(self, expression: Expression):
         self.expression = expression
 
+    def __str__(self):
+        return f"Negation(expression={self.expression})"
```

Comment:
Why not negation expression, to be consistent with the others?

Reply:
Fixed